src/search/landmarks/landmark_factory_h_m.h

```diff
@@ -101,7 +93,10 @@ class LandmarkFactoryHM : public LandmarkFactory {
     void print_fluentset(const VariablesProxy &variables, const FluentSet &fs);
     void print_pm_op(const VariablesProxy &variables, const PMOp &op);
 
```

Comment:
I see no reason why not.

Reply:
I was unsure because different parts of the code seem to do this differently, but I agree that we should declare things as constant if they are constant.

Reply:
I cannot see which code this discussion refers to, but it seems to be resolved anyway.

Reply:
I know that I once had a discussion about this with Jendrik, but I forgot the outcome/the general opinion, so this is fine with me.